ui/installer/vCenterForWindows/utils/WinSCP.ini

```diff
@@ -0,0 +1,294 @@
+[Configuration]
+
+[SshHostKeys]
+rsa2@22:localhost=0x23,0xba94d0c3d3a055146b387c820107b25998251e63846c28aa7693ce06aa4b7b05aeaaf31b3e47e8b274e0a7b90ab3eb2793b86a99f4c5f6cbf51b27c90d25242422a357860ef425616923c7281742408d26fbe408b92b9e8b63d40102f61a0b85fdd3f48f5ba3fdec30712b4161a9b589ad471c712e36be0d5e6f7606610ad703

```

Comment:
This is used by Windows version of the installer for copying plugin bundles to the cache folder. SshHostKeys is created during runtime but I will remove that and reflect that in the next commit